docker-compose.yml

```diff
@@ -65,7 +65,7 @@ services:
   # Same etcd is used for all services, but with different namespace
   etcd:
     hostname: etcd
-    image: docker.io/bitnami/etcd:3.5.5-debian-11-r16
+    image: docker.io/bitnami/etcd:3.5.11-debian-11-r3
```

Comment:
@hosekpeter 

Please update also Kubernetes configuration, in `provisioning` directory, search for `tag: 3.5.5-debian-11-r16`.

Then, the Templates API must be deployed to the testing stack, to validate it.